apis/node/identity.yaml

```diff
@@ -10,6 +10,9 @@ get:
       content:
         application/json:
           schema:
-            $ref: "../../beacon-node-oapi.yaml#/components/schemas/NetworkIdentity"
+            type: object
+            properties:
+              data:
+                $ref: "../../beacon-node-oapi.yaml#/components/schemas/NetworkIdentity"
```

Comment:
This looks as expected.
there's another PR with example value and type questions, but that's just FYI, and should be kept outside this change, since it's well defined.